.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "boot/efi"]
 	path = boot/efi
 	url = https://github.com/aurixos/efi
+[submodule "utils/kconfiglib"]
```

Comment:
I have 1 problem: submodules, it is a kinda bad solution, especially for kconfig

Reply:
@lolguy91 correct, even for the EFI library it should only be needed if the target ur compiling for needs EFI. Otherwise dont download it at all. I just used submodules as a quick test

Reply:
good

Reply:
I won't resolve until it gets fixed tho

Reply:
Targets would be: x86_64-dev-legacy or x86_64-dev-uefi. Legacy would depend on legacy bootloader and uefi would use the uefi bootloader (dev is instead of a board name or some shit. Kinda like clangs target naming)

Reply:
btw anyone can push to the repo

Reply:
might even do it myself tmrw

Reply:
> Targets would be: x86_64-dev-legacy or x86_64-dev-uefi. Legacy would depend on legacy bootloader and uefi would use the uefi bootloader (dev is instead of a board name or some shit. Kinda like clangs target naming)

 So instead of ` <arch><sub>-<vendor>-<sys>-<env>` it would be `<arch><sub>-<release>-<kind>` e.g:
 
 x86_64-dev-legacy. Note: kind can be other things.

Reply:
> > Targets would be: x86_64-dev-legacy or x86_64-dev-uefi. Legacy would depend on legacy bootloader and uefi would use the uefi bootloader (dev is instead of a board name or some shit. Kinda like clangs target naming)
> 
> So instead of ` <arch><sub>-<vendor>-<sys>-<env>` it would be `<arch><sub>-<release>-<kind>` e.g:
> 
> x86_64-dev-legacy. Note: kind can be other things.

armv5-dev-blablabla (double qoute reply is crazy)